### PR TITLE
Fix admin panel freeze with too large php-error log

### PR DESCRIPTION
**Pull request by Wiljami**
Abnormally long lines in the log file cause the admin panel to freeze. 

Now if too many chunks are generated while reading the log file, give up reading the log file to prevent the admin panel from freezing and return false like with too large files.

This does not fix the underlying issue that there is such log files though, and is a band-aid to stop the admin panel from freezing.

### CODE REVIEW

**Comment by Wiljami** [comment]
Since it is related to this topic I believe adding the feedback on log size errors to the users is within the this PR. The second commit adds feedback to the dashboard and the logs portion.

**Comment by Wiljami** [comment]
TODO: Translations once code is OK

File: modules/logs.php [hunk 1]
```diff
@@ -353,6 +363,9 @@ public static function read_log_lines_backwards( $filepath, $offset = -1, $lines
       // the newline in the end accouts for an extra line
       $lines--;
 
+      // Set max amount chunks to be read according to to the lines wanted from the log.
```
Comment (elguitar) [inline modules/logs.php:366]: typo

File: modules/login-notification.php [hunk 2]
```diff
@@ -192,6 +206,16 @@ public static function display_admin_errors_notification() {
       );
       echo '<div>' . $msg . '</div>';
     }
+    /**
+    * Display the amount of php-error.log lines that have appeared this week.
+    */
```
Comment (elguitar) [inline modules/login-notification.php:211]: This comment does not correspond to the code beneath

File: modules/logs.php [hunk 3]
```diff
@@ -210,6 +210,11 @@ public function render_log_view( $logfile, $regex = null, $max_num_of_rows ) {
       </div>
       <?php elseif ( ! $result ) : ?>
         <p><?php _e('Log empty', 'seravo'); ?></p>
+        <?php
+        // result -1 is the signal that something went wrong with reading the log
+        elseif ( $result === -1 ) :
+        ?>
+        <p><?php _e('Log is abnormally large and can not be displayed.', 'seravo'); ?></p>
```
Comment (elguitar) [inline modules/logs.php:217]: Should this be indentated?

File: modules/logs.php [hunk 4]
```diff
@@ -210,6 +210,11 @@ public function render_log_view( $logfile, $regex = null, $max_num_of_rows ) {
       </div>
       <?php elseif ( ! $result ) : ?>
         <p><?php _e('Log empty', 'seravo'); ?></p>
+        <?php
+        // result -1 is the signal that something went wrong with reading the log
+          elseif ( $result === -1 ) :
```
Comment (elguitar) [inline modules/logs.php:215]: This should not be indentated

**Pull request merged**